Dockerfile

```diff
@@ -1,13 +1,36 @@
 ARG ARCH="amd64"
 ARG OS="linux"
+FROM golang:alpine as builder
```

Comment:
This is not the way to do this. You should not modify the Dockerfile to do integration tests.

Reply:
I've modified the Dockerfile because building executable was off it. I think it is a good idea to show explicitly in your Dockerfile how to build your app. The first thing that I've tried was ```docker-compose up --build``` but it lacked the pgbouncer_exporter itself, then I've tried to build a container of with ```docker build -t pgbouncer-exporter:latest .``` and it failed because an executable was missing, so I've started to look through a make file in order to figure out how to build the exporter. So, I thought it would be a good idea to build and execute the whole exporter in a container with just one simple ```docker build -t pgbouncer-exporter:latest . ``` If there is an issue with doing so, I suggest I write README section on how to build pgbouncer_exporter from scratch.

Reply:
That is not how we build things.

If you want a good example of how we do testing, take a look at the integration tests we [do for the mysqld_exporter](https://github.com/prometheus/mysqld_exporter/blob/f5d5289f7627016d8b53b0b3fa7615da8a73b56a/.circleci/config.yml#L24-L41).